### DISCUSSION TITLE
Best method to migrate between PCs

### DISCUSSION THREAD

**Discussion by s38b35M5** [strawberrymusicplayer/strawberry#1385] [Q&A] [answered]
I have my library on my laptop, and recently obtained a more powerful desktop that I want to mirror everything to (playlists, stats, loudness scan, etc.)

What is the best way to perform this migration/transfer to the new PC?

**Comment by jonaski** [comment, marked as answer]
You can just copy the ~/.local/share/strawberry and ~/.config/strawberry directories.

**Reply by s38b35M5** [reply]
So as follows:

1. copy music library
2. copy folders `~/.local/share/strawberry` and `~/.config/strawberry`
3. launch smp and enjoy?

Apologies if that was in the FAQ/wiki and I missed it. If it's not, would it be possible to add for others to find?
